R/utilities.R

```diff
@@ -43,3 +43,14 @@ ifNotNull <- function(condition, outputIfNotNull, outputIfNull = NULL) {
     outputIfNull
   }
 }
+
+#' This is required to ensure that we have no issue using the mu symbol in different OS
+#' See https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/476 for details
+#' @import stringr
+encodeUnit <- function(unit) {
+  mu <- ospsuiteEnv$muSymbol
+  unit <- enc2utf8(unit)
+  unit <- str_replace(unit, rawToChar(as.raw(c(0xce, 0xbc))), mu)

```

Comment:
@Yuri05 Copy paste from your code here. I have no idea what this does , nor do I want to know ah ha